content/07.pathogenesis.md

```diff
@@ -9,6 +9,11 @@ This section would also be great for the introduction of zoonotic diseases which
 [How does SARS-CoV-2 enter human cells?]
 [What cells are primary infection sites for SARS-CoV-2?]
 [What structural aspects allow for viral entry]?
+ACE2 receptors have been identified as the binding site for SARS-CoV-2 in animals and humans [@doi:10.1038/nature02145;@doi:10.1007/s00134-020-05985-9;doi:10.1128/JVI.00127-20]. 
+The virus invades host cells by using ACE2 for entry and the serine protease TMPRSS2 for S protein priming. 
+It then fuses with the membrane and enters the cell via endosomes [@doi:10.1016/j.cell.2020.02.052]. 
+The C-terminal domain of SARS-CoV-2 S protein was identified as the key region in that interacts with the human hACE2 receptor. 
```

Comment:
```suggestion
The C-terminal domain of the SARS-CoV-2 S protein was identified as the key region of the virus that interacts with the human ACE2 (hACE2) receptor. 
```
I looked at the phrasing in the paper, it is a bit vague! Do you think this works?